### ISSUE TITLE
DeArrow Crash

### ISSUE THREAD

**Issue by DanielProg39** [libre-tube/LibreTube#5987]
### Steps to reproduce

Open the video from another app using a link.

### Expected behavior

Video should play normally.

### Actual behavior

App crashes.

### LibreTube version

LibreTube Debug, commit 096ab60

### Android version

Android 13

### Other details

This happened to me only once, actually. The second time I opened the video, it played. It definitely has something to do with DeArrow but the video does not have an alternative title. Here's the error:

```
java.lang.NullPointerException
 at com.github.libretube.util.DeArrowUtil.deArrowStreamItems(DeArrowUtil.kt:67)
 at com.github.libretube.util.DeArrowUtilKt.deArrowStreamItems(DeArrowUtil.kt:108)
 at com.github.libretube.ui.fragments.SubscriptionsFragment$loadNextFeedItems$1.invokeSuspend(SubscriptionsFragment.kt:220)
 at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
 at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
 at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
 at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
 at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
 at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
 at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
 at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
 Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@7589507, Dispatchers.IO]

```

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
